pages/apadrinhamento.js

```diff
@@ -0,0 +1,41 @@
+import Layout from 'components/Layout/Layout'
+import PadrinhoContent from 'components/Padrinho/PadrinhoContent'
+import Button from 'components/Button/Button'
+import { getData } from 'functions/getData';
+
+const Apadrinhamento = (props) => {
+  const { headerMenu } = props
+
+  return (
+    <Layout pageTitle={`Torne-se um Padrinho`} {...props}>
+      <PadrinhoContent title={`Apadrinhe um recém-chegado ao Canada!`} />
+      <section className={`text-center`}>
```

Comment:
Move this section to the component itself. You can split in small components there, but keep it on the same context.

Reply:
Is that ok if I just get rid of the section of buttons and keep everything within its component? Just want to make it cleaner

Reply:
yes

Reply:
done